src/astro/sql/operators/merge.py

```diff
@@ -37,7 +36,7 @@ def __init__(
         self.target_conflict_columns = target_conflict_columns
         self.source_to_target_columns_map = source_to_target_columns_map
         self.if_conflicts = if_conflicts
-        task_id = task_id or get_unique_task_id(self.target_table.name + "_merge")
+        task_id = task_id or "merge"
```

Comment:
What would happen if this task is run multiple times without defining the `task_id`? Does `task_id` need to be unique?

Reply:
I have added `get_unique_task_id("_merge")` to the `task_id` in the [commit](https://github.com/astronomer/astro-sdk/pull/423/commits/605f7e9a8703ec179d296a2e07a4f8cfc4d61cb7).